pkg/audit/manager.go

```diff
@@ -26,7 +26,7 @@ var log = logf.Log.WithName("controller").WithValues("metaKind", "audit")
 
 const (
 	crdName       = "constrainttemplates.templates.gatekeeper.sh"
-	constraintsGV = "constraints.gatekeeper.sh/v1alpha1"
+	constraintsGV = "constraints.gatekeeper.sh/v1beta1"
```

Comment:
What if users have both `v1alpha1` and `v1beta1` constraints in the system? This update will only look for the `v1beta1` constraints.

Reply:
Story time :)

This isn't an issue, as all constraints are conceptually simultaneously both alpha and beta at the same time. If a user writes a constraint, it gets stored at the current storage version, when constraints are read, they are read at the requested version.

However...

While verifying this, I found a separate issue. There is a bug in multi-version constraints that when:

1. Admission Webhooks are enabled and registered (GK is an admission webhook)
2. A cluster has a CR persisted in an old storage version
3. A cluster upgrades the CRD and bumps the storage version

It is impossible to write a beta version to a resource persisted in alpha. The API server tries to do a (not-unstructured) scheme conversion and errors out. Fortunately it's possible to bump the storage version by writing an alpha version. I've added some code to do that on startup in `pkg/upgrade`.

Reply:
Posted bug to k8s repo:

https://github.com/kubernetes/kubernetes/issues/80494